lib/python_models.py

```diff
@@ -375,56 +425,10 @@ def setupmodel(model, ind):
         parname.insert(ind.heqpm,   'Hook, +/-')
         parname.insert(ind.heqper,  'Hook, Period')
         #DEFINE RAMP MODEL
+ #       myfuncs.append(heqramp)
         myfuncs.append(mc.heqramp)
         saveext.append('heq')
         functype.append('ramp')
-    elif model[i] == 'zhouramp':
```

Comment:
Let's keep `zhouramp` and `recte`. They were deleted when I sent you the new version of this file that came from POET

Reply:
also added them back into `lib/paramedit.py`